doc/README.md

```diff
@@ -3335,6 +3335,7 @@ additional arguments.
 #### Arguments
 1. `func` *(Function)*: The function to cap arguments for.
 2. `[n=func.length]` *(number)*: The arity cap.
+3. `[exact=false]` *(boolean)*: Create a wrapper with the exact arity.
```

Comment:
You don't have to update the doc/README.md, it's generated from the lodash.src.js at a later time.